db/migrate/20201202164337_control_explorer_split_feature_mapping.rb

```diff
@@ -0,0 +1,84 @@
+    "profile_new"            => "miq_policy_set_new",
+    "profile_delete"         => "miq_policy_set_delete",
+    "profile_edit"           => "miq_policy_set_edit",
+    "policy"                 => "miq_policy",
+    "policy_view"            => "miq_policy_view",
+    "policy_admin"           => "miq_policy_admin",
+    "policy_new"             => "miq_policy_new",
+    "policy_copy"            => "miq_policy_copy",
+    "policy_edit"            => "miq_policy_edit",
+    "policy_edit_conditions" => "miq_policy_conditions_assignment",
+    "policy_edit_events"     => "miq_policy_events_assignment",
+    "policy_delete"          => "miq_policy_delete",
+    "event"                  => "miq_event",
+    "action"                 => "miq_action",
+    "action_admin"           => "miq_action_admin",
+    "action_show_list"       => "miq_action_show_list",
+    "action_show"            => "miq_action_show",
+    "action_new"             => "miq_action_new",
+    "action_edit"            => "miq_action_edit",
+    "action_delete"          => "miq_action_delete",
+    "alert_profile"          => "miq_alert_set",
+    "alert_profile_admin"    => "miq_alert_set_admin",
+    "alert_profile_new"      => "miq_alert_set_new",
+    "alert_profile_edit"     => "miq_alert_set_edit",
+    "alert_profile_assign"   => "miq_alert_set_assign",
+    "alert_profile_delete"   => "miq_alert_set_delete",
+    "alert"                  => "miq_alert",
+    "alert_admin"            => "miq_alert_admin",
+    "alert_new"              => "miq_alert_new",
+    "alert_copy"             => "miq_alert_copy",
+    "alert_edit"             => "miq_alert_edit",
+    "alert_delete"           => "miq_alert_delete",
+  }.freeze
+
+  def up
+    return if MiqProductFeature.none?
+
+    say_with_time 'Renaming old hidden features under control explorer after the split' do
+      new_feature = MiqProductFeature.find_or_create_by!(:identifier => 'control')
+      user_role_ids_with_old_features = MiqRolesFeature.where(:miq_product_feature_id => old_feature_ids).pluck(:miq_user_role_id).uniq if old_feature_ids
+      user_role_ids_with_old_features.each do |user_role_id|
+        MiqRolesFeature.create!(:miq_product_feature_id => new_feature.id, :miq_user_role_id => user_role_id)
+      end
+
+      # Direct renaming of features
+      FEATURE_MAPPING.each do |from, to|
+        MiqProductFeature.find_by(:identifier => from)&.update!(:identifier => to)
+      end
+    end
+  end
+
+  def down
+    return if MiqProductFeature.none?
+
+    say_with_time 'Naming features back to match old features before the explorer split' do
+      admin_feature = MiqProductFeature.find_or_create_by!(:identifier => 'control_explorer')
+
+      MiqRolesFeature.where(:miq_product_feature_id => admin_feature.id).each do |feature|
+        old_feature_ids.each do |id|
+          MiqRolesFeature.create!(:miq_product_feature_id => id, :miq_user_role_id => feature.miq_user_role_id)
```

Comment:
If we are creating in the up I'd expect a delete in the down, but I don't see any